.github/workflows/run-match.yml

```diff
@@ -52,12 +52,12 @@ jobs:
           # Convert the sleep comments to print statement so there's a way to
           # gauge progress and so that GitHub doesn't shut things down due to
           # lack of output (the latter is a theoretical concern).
-          sed -r 's/# sleep(.*)/print("sleep\1")/' ../controllers/example_controller/example_controller.py > robot.py
+          sed -r 's/# sleep(.*)/print("sleep\1")/' ../controllers/example_controller/example_forklift.py > forklift.py
```

Comment:
How come we're only doing these "hacks" for just the forklift?

Reply:
The previous CI only tested a robot in zone 1 so this just makes that only the forklift in zone 1. This should probably be expanded to include all 4 robots.

Reply:
We now test all 4 robots in CI